pkg/commands/git_commands/working_tree.go

```diff
@@ -213,9 +213,18 @@ func (self *WorkingTreeCommands) DiscardUnstagedFileChanges(file *models.File) e
 	return self.cmd.New("git checkout -- " + quotedFileName).Run()
 }
 
+func (self *WorkingTreeCommands) Omit(filename, destination string) error {
```

Comment:
I don't think we need this function: using `self.os.AppendLineToFile` directly is fine.

Reply:
removed :)